src/plone/restapi/serializer/dxcontent.py

```diff
@@ -125,9 +130,7 @@ def __call__(self, version=None, include_items=True):
         if target_url:
             result["targetUrl"] = target_url
 
-        result["allow_discussion"] = getMultiAdapter(
-            (self.context, self.request), name="conversation_view"
-        ).enabled()
+        result["allow_discussion"] = get_allow_discussion_value(self.context, self.request)
```

Comment:
The assignment of `result["allow_discussion"]` must be conditional. see my suggestion at:

https://github.com/plone/plone.restapi/pull/1655#discussion_r1239804188

```suggestion
        get_allow_discussion_value(self.context, self.request, result)
```